documentation/original-METS/README.md

```
@@ -0,0 +1,24 @@
+
+## **Functionality**
+  - When a logical AIP is created, a METS file will also be created.
+  - When a SIP is ingested, the AIP will retain the original METS file(s) from the SIP.
+  - In the ingested METS files some attributes are updated so they comply with the DILCIS Board.
+  - At each archival preservation action a PREMIS file is created, and that file is recorded in the related METS file.
+  - When an AIP is ingested and moved, the nearest relationships are documented in the METS files for both parent and child.
+
+
+## **System overview**
+
+#### The original-METS has to know the main-storage-path.
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)